bin/oref0-online.sh

```diff
@@ -43,13 +64,19 @@ main() {
             #restart_networking
         fi
     fi
-    echo Finished oref0-online.
+    echo Finished oref0-online. 
+}
+
+function print_bluetooth_name {
+    echo -n "At $(date), my bluetooth is connected to "
+    echo -n ${MACs}
```

Comment:
This won't support configurations that let the rig pair with multiple BT phones.  MACs is a list of all MAC addresses to try.  If we want to print_bluetooth_name we need to figure out how to actually print which device it's connected to.

Reply:
How have you been running with multiple Bluetooth Hotspots? I was thinking that would be nice. since I carry around two phones. and well yeah we all sometimes forget to leave it one bluetooth hotspot. 
I was looking into having it say the device name that it's connected too, but only way I can find even to see that is to go into bluetoothctl. would be nice to have it do that though

Reply:
I run with multiple Bluetooth connections by putting the Mac addresses in serially in the oref0-online cron line, separated by a space.

Reply:
Same. I changed this to actually print the name(s) of the phone(s) it’s connected to instead.

Reply:
Does this require a reinstall to be made to work or will it work off a standard pull and npm global install in the dev branch?

Reply:
If you have BT hotspot enabled it will mostly work without re-running oref0-setup, but there are some modifications there to rc.d that will change the behavior you get when restarting the rig.  @Bender1061 had quite a few issues with his rig getting stuck in hotspot mode before he made those changes.